virtual-organizations/OSG.yaml

```diff
@@ -136,10 +136,11 @@ DataFederations:
               Map Subject: True
         AllowedOrigins:
           - CHTC_OSPOOL_ORIGIN
+          - CHTC-ap40
         AllowedCaches:
           - ANY
-        Writeback: https://origin-auth2001.chtc.wisc.edu:1095
-        DirList: https://origin-auth2001.chtc.wisc.edu:1095
+        Writeback: https://ospool-ap2140.chtc.wisc.edu:8443
+        DirList: https://ospool-ap2140.chtc.wisc.edu:8443
```

Comment:
You should leave this for a separate PR: we'll need to merge this one to be able to generate the authfile/scitokens config for ap2140 but we shouldn't change the Writeback and DirList until we're ready to switch everything over.

Reply:
We're going to pull this entirely since we're going to need a flag day